torch/onnx/_internal/fx/passes/decomp.py

```diff
@@ -32,6 +37,30 @@ def _rename_placeholder_targets(
     module.recompile()
 
 
+def wrapped_fn(fn: Callable, mode: str, allow_non_fake_inputs: bool, args: List[Any]):
+    context = enable_python_dispatcher if mode == "symbolic" else contextlib.nullcontext
+    shape_env = ShapeEnv() if mode == "symbolic" else None
+    fake_mode = torch._dynamo.utils.fake_mode_from_tensors(args)
+    if mode is None:
```

Comment:
```suggestion
    if fake_mode is None:
```
I think this resolves the onnx errors in CI. I'm not sure about the one you posted though, building locally to have a look.